Update Coveralls integration and badges

Old: https://coveralls.io/github/ntwb/eslint-plugin-wordpress/
New: https://coveralls.io/github/WordPress-Coding-Standards/eslint-plugin-wordpress

Coveralls fails to throw an error on a failed upload /shrug

Via https://travis-ci.org/WordPress-Coding-Standards/eslint-plugin-wordpress/jobs/251409045#L1379
```shell
$ npm run coveralls
> eslint-plugin-wordpress@0.1.0 coveralls /home/travis/build/WordPress-Coding-Standards/eslint-plugin-wordpress
> nyc report --reporter=text-lcov | coveralls
/home/travis/build/WordPress-Coding-Standards/eslint-plugin-wordpress/node_modules/coveralls/bin/coveralls.js:18
        throw err;
        ^
Bad response: 422 {"message":"Couldn't find a repository matching this job.","error":true}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! eslint-plugin-wordpress@0.1.0 coveralls: `nyc report --reporter=text-lcov | coveralls`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the eslint-plugin-wordpress@0.1.0 coveralls script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2017-07-08T06_28_12_791Z-debug.log
Done. Your build exited with 0.
```

--- a/README.md
+++ b/README.md
@@ -2,7 +2,7 @@
 
 A collection of custom ESLint rules that help enforce JavaScript coding standard in the WordPress project.
 
-[![Build Status](https://api.travis-ci.org/WordPress-Coding-Standards/eslint-plugin-wordpress.svg?branch=master)](https://travis-ci.org/WordPress-Coding-Standards/eslint-plugin-wordpress) [![Coverage Status](https://coveralls.io/repos/github/ntwb/eslint-plugin-wordpress/badge.svg?branch=master)](https://coveralls.io/github/ntwb/eslint-plugin-wordpress?branch=master) [![bitHound Overall Score](https://www.bithound.io/github/WordPress-Coding-Standards/eslint-plugin-wordpress/badges/score.svg)](https://www.bithound.io/github/WordPress-Coding-Standards/eslint-plugin-wordpress) [![NPM version](http://img.shields.io/npm/v/eslint-plugin-wordpress.svg)](https://www.npmjs.org/package/eslint-plugin-wordpress)
+[![Build Status](https://api.travis-ci.org/WordPress-Coding-Standards/eslint-plugin-wordpress.svg?branch=master)](https://travis-ci.org/WordPress-Coding-Standards/eslint-plugin-wordpress) [![Coverage Status](https://coveralls.io/repos/github/WordPress-Coding-Standards/eslint-plugin-wordpress/badge.svg?branch=master)](https://coveralls.io/github/WordPress-Coding-Standards/eslint-plugin-wordpress?branch=master) [![bitHound Overall Score](https://www.bithound.io/github/WordPress-Coding-Standards/eslint-plugin-wordpress/badges/score.svg)](https://www.bithound.io/github/WordPress-Coding-Standards/eslint-plugin-wordpress) [![NPM version](http://img.shields.io/npm/v/eslint-plugin-wordpress.svg)](https://www.npmjs.org/package/eslint-plugin-wordpress)
 
 ## Installation
 

